packages/expo-splash-screen/ios/EXSplashScreen/EXSplashScreenViewNativeProvider.m

```diff
@@ -0,0 +1,30 @@
+// Copyright © 2018 650 Industries. All rights reserved.
+
+#import <EXSplashScreen/EXSplashScreenViewNativeProvider.h>
+#import <UMCore/UMLogManager.h>
+
+@implementation EXSplashScreenViewNativeProvider
+
+- (nonnull UIView *)createSplashScreenView
+{
+  @try {
+    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"SplashScreen" bundle:[NSBundle mainBundle]];
+    UIViewController *splashScreenViewController = [storyboard instantiateViewControllerWithIdentifier:@"SplashScreenViewController"];
+    UIView *splashScreenView = splashScreenViewController.view;
+    return splashScreenView;
+  } @catch (NSException *_) {
+    UMLogInfo(@"Expo SplashScreen.storyboard is missing.");
+  }
+  
+  @try {
+    NSArray *views = [[NSBundle mainBundle] loadNibNamed:@"SplashScreen" owner:self options:nil];
+    UIView *splashScreenView = views.firstObject;
```

Comment:
Is it possible that `views.firstObject` is `nil`? If so, I guess we should throw an exception here.